src/Lattice/Uniform2DGridLayout.cpp

```diff
@@ -46,54 +42,54 @@ void Uniform2DGridLayout::makeShell(std::vector<SingleParticleIndex_t>& RS,
                                     const SingleParticleIndex_t& nc,
                                     value_type rr)
 {
-  std::map<int,std::vector<SingleParticleIndex_t>*> rs;
-  std::map<int,bool> cb;
-  Scalar_t scaleL = static_cast<value_type>(8*NP[0]*NP[1]);
-  int ic=0,ih;
-  Scalar_t dx=Delta[0]*0.5;
-  Scalar_t dy=Delta[1]*0.5;
-  for(int ix1=-nc[0]; ix1<=nc[0]; ix1++)
+  std::map<int, std::vector<SingleParticleIndex_t>*> rs;
+  std::map<int, bool> cb;
+  Scalar_t scaleL = static_cast<value_type>(8 * NP[0] * NP[1]);
+  int ic          = 0, ih;
+  Scalar_t dx     = Delta[0] * 0.5;
+  Scalar_t dy     = Delta[1] * 0.5;
+  for (int ix1 = -nc[0]; ix1 <= nc[0]; ix1++)
   {
-    Scalar_t x= std::abs(ix1)*Delta[0];
-    for(int ix2=-nc[1]; ix2<=nc[1]; ix2++)
+    Scalar_t x = std::abs(ix1) * Delta[0];
+    for (int ix2 = -nc[1]; ix2 <= nc[1]; ix2++)
     {
-      if(ix1==0 && ix2 == 0)
+      if (ix1 == 0 && ix2 == 0)
         continue;
-      Scalar_t y=std::abs(ix2)*Delta[1];
+      Scalar_t y = std::abs(ix2) * Delta[1];
       ic++;
-      SingleParticlePos_t tc(x-dx,y-dy);
-      int ih = static_cast<int>(Dot(tc,tc)*scaleL);
-      std::map<int,std::vector<SingleParticleIndex_t>*>::iterator it = rs.find(ih);
-      if(it == rs.end())
+      SingleParticlePos_t tc(x - dx, y - dy);
+      int ih                                                          = static_cast<int>(Dot(tc, tc) * scaleL);
+      std::map<int, std::vector<SingleParticleIndex_t>*>::iterator it = rs.find(ih);
+      if (it == rs.end())
       {
         std::vector<SingleParticleIndex_t>* ns = new std::vector<SingleParticleIndex_t>;
-        ns->push_back(SingleParticleIndex_t(ix1,ix2));
+        ns->push_back(SingleParticleIndex_t(ix1, ix2));
         rs[ih] = ns;
       }
       else
       {
-        (*it).second->push_back(SingleParticleIndex_t(ix1,ix2));
+        (*it).second->push_back(SingleParticleIndex_t(ix1, ix2));
       }
     }
   }
   RS.resize(ic);
   FS.resize(ic);
-  int irc = static_cast<int>(4.0*rr*scaleL);
-  int ir=0;
-  std::map<int,std::vector<SingleParticleIndex_t>*>::const_iterator cit = rs.begin();
-  while(cit != rs.end())
+  int irc                                                                = static_cast<int>(4.0 * rr * scaleL);
+  int ir                                                                 = 0;
```

Comment:
Alignment here is a bit extreme.  Perhaps not a big deal if infrequent.

Reply:
It is due to the next line with a = very far away.
Currently clang-format is not smart enough to handle this.
Hopefully they will introduce some heuristics or machine-learning algorithm to improve this.

Reply:
adding a blank line fixes this.  For now clang-format doesn't provide a penalty weighting for this particular reformat so this is what we can do.